modules/vulnerabilities/unix/webapp/moinmoin_195/secgen_metadata.xml

```diff
@@ -17,6 +17,23 @@
   <access>remote</access>
   <platform>linux</platform>
 
+  <read_fact>strings_to_leak</read_fact>
+  <read_fact>site_name</read_fact>
+  <read_fact>default_page</read_fact>
+
+  <default_input into="strings_to_leak">
+    <value>This is some leaked data.</value>
+    <generator type="string_generator"/>
```

Comment:
I think it helps to be more specific that, as you may end up with a random string that they could spend hours trying to crack, better to generate messages, etc